tests/test_gcc.py

```diff
@@ -13,7 +13,7 @@
 
 CONTAINERFILE_HELLO = f"""
 WORKDIR /src
-RUN curl -sLO https://ftpmirror.gnu.org/hello/hello-{_HELLO_VERSION}.tar.gz && \\
+RUN curl -sLO https://ftp.gnu.org/gnu/hello/hello-{_HELLO_VERSION}.tar.gz && \\
```

Comment:
you're just changing the mirror redirector to a hardcoded site. the webpage explicitly recommends against this practice. 

the redirector is some times a bit flakey, it will recover. the change is imho not needed.